lib/engine/game/g_1860/game.rb

```diff
@@ -743,8 +743,8 @@ def current_layer
         end
 
         def move_marker_to_bottom!(corporation)
-          @stock_market.move_left(corporation)
-          @stock_market.move_right(corporation)
+          @stock_market.move_down(corporation)
+          @stock_market.move_up(corporation)
```

Comment:
While I don't disagree with this change, it shouldn't have caused a failure (assume a move_right still reverses a move_left).

Reply:
I didn't know what the behavior was doing, so I changed all of them. Now that you explained it, I agree it works either way.